src/main/kotlin/no/nav/familie/ef/søknad/api/dto/søknadsdialog/SøknadSkolepengerDto.kt

```diff
@@ -8,5 +8,6 @@ data class SøknadSkolepengerDto(
         val søkerBorPåRegistrertAdresse: BooleanFelt?,
         val utdanning: UnderUtdanning,
         val dokumentasjonsbehov: List<Dokumentasjonsbehov>,
-        val locale: String = "nb"
+        val locale: String = "nb",
+        val skalBehandlesINySaksbehandling: Boolean = true,
```

Comment:
Burde default vær false?

Reply:
Altså:  default er at vi prøver å lage sak i infotrygd (som i prod i dag)
Vi legger jo inn en "opt out" for dette for å kunne behandle disse søknadene i vår løsning? 
(men begge deler vil jo fungere - bare litt rart om denne kommer inn som true til mottak for alle også i prod)

Reply:
Det er helt riktig! :) Ble feil i refakturering